cmd/gke-exec-auth-plugin/cloudbuild.yaml

```diff
@@ -0,0 +1,13 @@
+steps:
+- name: 'gcr.io/cloud-builders/bazel'
+  args: ['build', '--', '//...', '-//vendor/...']
+- name: 'gcr.io/cloud-builders/bazel'
+  args: ['test', '--test_output=errors', '--', '//...', '-//vendor/...']
+- name: 'gcr.io/cloud-builders/gsutil'
+  args: ['-m', 'cp', 'bazel-bin/cmd/gke-exec-auth-plugin/linux_amd64_stripped/gke-exec-auth-plugin', 'gs://awly-dev/gke-exec-auth-plugin/$_PLUGIN_VER/gke-exec-auth-plugin']
+timeout: 1200s
+options:
+  machineType: 'N1_HIGHCPU_8'
+  substitutionOption: 'MUST_MATCH'
+substitutions:
+  _PLUGIN_VER: v0.0.1
```

Comment:
how should versioning work for development builds?

Reply:
For dev builds you just `bazel build //cmd/gke-exec-auth-plugin`.
This versioning is for releases only.